apphomepage/templates/apphomepage/homepage.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="apple-touch-icon" sizes="180x180" href="favicon/apple-touch-icon.png">
+<link rel="icon" type="image/png" sizes="32x32" href="favicon/favicon-32x32.png">
+<link rel="icon" type="image/png" sizes="16x16" href="favicon/favicon-16x16.png">
+<link rel="manifest" href="favicon/site.webmanifest">
+<link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
+<meta name="msapplication-TileColor" content="#da532c">
+<meta name="theme-color" content="#ffffff">
+<title>pickATrip</title>
+</head>
+
+<body>
+	<h1>Pick a trip</h1>
+	<h2>picATrip is a web platform for sharing experiences from trips in a form of photos</h2>
```

Comment:
Perhaps something like this would be a more accurate description:

picATrip is a web platform where users can suggest trips, share their experiences and photos and discuss them with friends.

I still think the group should discuss this together (also in the context of the ReadMe), although I might be in the minority here, judging by the discussion on Slack.